index.d.ts

```diff
@@ -44,6 +44,21 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * Hook that calls inputHandler callback with input that program received.
+ * Additionally contains helpful metadata for detecting when arrow keys were pressed.
+ */
+export function useInput(
+	inputHandler: (input: string, meta: Meta) => void
+): void;
+
+export interface Meta {
+	up: boolean
+	down: boolean
+	left: boolean
+	right: boolean
```

Comment:
Would be useful to also add `ctrl`, `meta`, `shift`. The original `keypress` event contains that.

https://thisdavej.com/making-interactive-node-js-console-apps-that-listen-for-keypress-events/